README.md

```diff
@@ -5,6 +5,68 @@ https://flutter.dev/
 Flutter in an open Source SDK for creating high-performance mobile apps for IOS and Android.
 The Flutter makes it easier for you to build user interfaces, while reducing the amount of code required to create and update your app.
 
+## Why use Flutter?
+
+- Flutter can be used to build applications for Android and IOS
+- The Dart language used in Flutter is quite similar to Java
+- We can update the application in real time
+- If we close the application when we open it again we can continue from where we were
+- Flutter benefits from several objects that help us to facilitate the construction process
+
+## Core Principles
+
+Flutter is designed for the creation of 2D mobile applications.</br> 
+It contains several widgets and tools that facilitate the realization of an app.</br>
+
+## Widgets
+
+Widget is a component that can be used to simplify the process of building applications.</br>
+It usually contains windows, buttons, icons, menus, scroll bars and other features.</br>
+
+A widget can define:
+- Physical elements of an application (buttons, menus or bars)
+- Visual elements such as colors  
+- Elements of the application aspect such as edges or the padding of the screen
+
+Widgets form a hierarchy whenever we use one of them we will have another that will complement it.</br>
+There are widgets that are used in all applications and some that are used in specific situations.</br>
+
+![Screen Shot 2020-02-06 at 19 01 17](https://user-images.githubusercontent.com/27420533/73969408-4475d280-4913-11ea-8384-99c863321155.png)
```

Comment:
Good image showing the _hierarchy_ of widgets. Did you _create_ the diagram?
If you didn't _create_ it from scratch, please _attribute_ it to the source where you got it.

e.g: this gif explains the Flutter widget hierarchy visually:
![flutter-widgets-gif](https://res.cloudinary.com/practicaldev/image/fetch/s--gCy5MuRs--/c_limit%2Cf_auto%2Cfl_progressive%2Cq_66%2Cw_880/https://thepracticaldev.s3.amazonaws.com/i/q0m1c1s77u7uky4zwh2m.gif)
See: https://dev.to/rubensdemelo/flutter-widget-tree-and-state-management-31an

Reply:
Wow is a really good and different illustration of how Flutter works.
Super interesting a diagram like this. 🆒 😃